circle.yml

```diff
@@ -1,10 +1,10 @@
 machine:
+  node:
+    version: 6.9.1
```

Comment:
6.9.1 is the latest stable and Circle CI doesn't complain about it. For some reason I was having issues with spread syntax. This change will also happen when we adopt selenium-webdriver 3.